readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: wordpressdotorg, audrasjb, whodunitagency, pbiron, whyisjake, xkon
 Requires at least: 5.3
 Tested up to: 5.4
 Requires PHP: 5.6
-Stable tag: 0.7.0
+Stable tag: 0.8.0
```

Comment:
Need to also update the version number in `package.json` :-)

Reply:
and package-lock.json

Reply:
on it!

Reply:
updated